wp-config-pantheon.php

```diff
@@ -45,6 +45,11 @@
 define('SECURE_AUTH_SALT', $_ENV['SECURE_AUTH_SALT']);
 define('LOGGED_IN_SALT', $_ENV['LOGGED_IN_SALT']);
 define('NONCE_SALT', $_ENV['NONCE_SALT']);
+
+// When developing a WordPress Multisite locally, ensure that this constant is set.
+if (getenv('FRAMEWORK') == 'wordpress-multisite') {
```

Comment:
```suggestion
if (getenv('FRAMEWORK') == 'wordpress_network') {
```
Framework name is `wordpress_network`